.cargo/config.toml

```diff
@@ -1,9 +1,9 @@
 # Choose a default "cargo run" tool.
 # probe-run is recommended if you have a debugger
-# elf2uf2-rs loads firmware over USB when the rp2040 is in boot mode
+# elf2uf2-rs loads firmware over USB when the rp2040 is in boot mode, which required you to install corresponding package by executing "$ cargo install elf2uf2-rs"
```

Comment:
I don't think this comment is needed - both probe-run and elf2usb2-rs require installation.

Reply:
I've added this to the README in #14